mlir/test/Dialect/Vector/test-scalable-upper-bound.mlir

```diff
@@ -0,0 +1,137 @@
+// RUN: mlir-opt %s -test-affine-reify-value-bounds -cse -verify-diagnostics \
+// RUN:   -verify-diagnostics -split-input-file | FileCheck %s
```

Comment:
Many of these tests don't make sense unless you know that `vscale` is within the `(1, 16)` range and IMHO that information should be available in this test file. Either as a comment or, better, a flag (or a couple of flags) passed to `mlir-opt`. WDYT?

Reply:
I've updated the `test.reify_scalable_bound` op so the vscale range must be explicitly set (before it defaulted to 1 to 16). So the ops now look like:
```
%bound = "test.reify_scalable_bound"(%vscale_plus_c8) {type = "UB", vscale_min = 1, vscale_max = 16} : (index) -> index
```